lib/db2.js

```diff
@@ -88,6 +90,30 @@ DB2.prototype.update = function(model, where, data, options, cb) {
 };
 
 /**
+ * Update all instances that match the where clause with the given data
+ *
+ * @param {string} model The model name
+ * @param {Object} where The where object
+ * @param {Object} data The property/value object representing changes
+ * to be made
+ * @param {Object} options The options object
+ * @param {Function} cb The callback function
+ */
+DB2.prototype.replaceById = function(model, where, data, options, cb) {
+  debug('DB2.prototype.replaceById (enter): where=%j, data=%j, options=%j\n',
+        where, data, options);
+  var self = this;
+  var stmt = self.buildReplace(model, where, data, options);
+  var id = self.idName(model);
+  var sql = 'SELECT \"' + id + '\" FROM FINAL TABLE (' + stmt.sql + ')';
+  self.execute(sql, stmt.params, options, function(err, info) {
+    if (cb) {
+      cb(err, {count: info.length});
```

Comment:
There appear to be no tests that validate the return value and none of the function headers describe what data is expected to be returned.  Do we have such info and I will change this to return the appropriate info?